test/content/schedule.test.js

```diff
@@ -129,6 +129,24 @@ describe('Content Schedule', () => {
     expect(matchers.seasonalGear.end).to.eql(moment.utc(`2025-03-21T${String(switchoverTime).padStart(2, '0')}:00:00.000Z`).toDate());
   });
 
+  it('uses correct date for first hours of the month', () => {
+    // if the date is checked before CONTENT_SWITCHOVER_TIME_OFFSET,
+    // it should be considered the previous month
+    const date = new Date('2024-05-01T02:00:00.000Z');
+    const matchers = getAllScheduleMatchingGroups(date);
+    expect(matchers.petQuests.items).to.contain('snake');
+    expect(matchers.petQuests.items).to.not.contain('horse');
+  });
+
+  it('uses correct date after switchover time', () => {
+    // if the date is checked after CONTENT_SWITCHOVER_TIME_OFFSET,
+    // it should be considered the current
+    const date = new Date('2024-05-01T09:00:00.000Z');
+    const matchers = getAllScheduleMatchingGroups(date);
+    expect(matchers.petQuests.items).to.contain('snake');
+    expect(matchers.petQuests.items).to.not.contain('horse');
```

Comment:
The expectations here are the same, between the "previous month" and "current month" test cases. Is that right? I thought the idea would be that the content would change over?

Reply:
It is intentional that they are the same, yes. The content does change over, however the pet quests don't. I could also add a case that checks against content that does change on the 1st, however this also is already sufficient to test against it using the right date. (since the bug was that the incorrect date was the last day of the current month instead of the previous month)

Reply:
I do think it'd be a good idea to have `expect`s about things that both should and shouldn't change, for coverage and clarity!